fmskill/compare.py

```diff
@@ -676,10 +802,7 @@ def scatter(
             plt.plot([xlim[0], xlim[1]], [xlim[0], xlim[1]], label="1:1", c="blue")
             plt.plot(xq, yq, label="Q-Q", c="gray")
             plt.plot(
-                x,
```

Comment:
The reason why these are going back and forward when each of us saves is because of the different versions of black we use. @ecomodeller and I use version 20.8. @Hendrik1987 could you please update to version too? 

    > conda install -c conda-forge black

Reply:
Done. Can you specify this in `setup.py`?

Reply:
Yes, good idea - will do that